shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -654,14 +654,14 @@ - (instancetype)initWithTarget:(id)target
 
 - (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
     shouldBeRequiredToFailByGestureRecognizer:(UIGestureRecognizer*)otherGestureRecognizer {
-  // The forwarding gesture recognizer should always get all touch events, so it should not be
-  // required to fail by any other gesture recognizer.
-  return otherGestureRecognizer != _forwardingRecognizer && otherGestureRecognizer != self;
```

Comment:
Since we set self as the delegate, `otherGestureRecognizer != self` is always YES.